dds2val/README.md

```diff
@@ -41,6 +41,7 @@ Configuration for the DDS provider is solved through setting environment variabl
 | DAPR_GRPC_PORT                | None          | If [DAPR](https://dapr.io/) gets used port of DAPR Sidecar. Overwrites VDB_PORT variable                                                         |
 | MAPPING_FILE                  | mapping.yml   | Place of mapping file from DDS to VSS                                                                                                            |
 | VDB_PORT                      | 55555         | On which port the (vehicle) databroker is expected. If you want to use DAPR use DAPR_GRPC_PORT.                                                  |
+| TOKEN                         | None          | JWT token which will get used to authorize to databroker                                                                                         |
```

Comment:
So here it must be a token, giving a file path to a token does not work, or? If so - would it be valuable with a "real example" on how to use/start dds-feeder with authentication, i.e. giving token as env-variable?